### PR TITLE
Add course description and page link

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Fix ##1179
#### What's this PR do?

Add description to courses in the program page. Make course name a link to course page on edx.

screen shot 2016-10-04 at 11 43 34 am

### CODE REVIEW

File: cms/api.py [hunk 1]
```diff
@@ -35,3 +37,15 @@ def get_course_enrollment_text(course):
...
+
+def get_course_url(course):
+    """
+    Construct the course page url
+    """
+    course_run = course.get_next_run()
+    if course_run:
+        course_key = course_run.edx_course_key
+        url = settings.EDXORG_BASE_URL
+        return "{}/courses/{}/about".format(url, course_key)
```
Comment (noisecapella) [inline cms/api.py]: Can you use `urljoin` here to join the edX base URL and the rest of the path? String concat is fine for the `courses/{}/about` part of the URL

**Comment by noisecapella** [comment]
Just the one comment, functionality looks fine

**Comment by annagav** [comment]
@noisecapella, addressed all comments!

**Comment by noisecapella** [comment]
:+1:

**Pull request merged**